mpl_interactions/generic.py

```diff
@@ -487,6 +487,8 @@ def _ipython_display_(self):
 
 def hyperslicer(
     arr,
+    is_rgb = False,
+    is_rbga = False,
```

Comment:
instead of two kwargs can we combine them into something like:  
`is_rgb_like`  
or  
`is_color_image`

Reply:
but then how do you differentiate between rgb and rgba? You need to know how many dims to pass to imshow so you can set up sliders for the rest of them.

Reply:
Also can you move this lower into the arguments. It maybe makes sense to keep the first several arguments the same as `plt.imshow`. that makes it maximally easy to transition if you are using positional args with imshow.
```python
plt.imshow(arr[0,1,1,:,:], 'viridis')
```
can become
```python
hyperslicer(arr, 'viridis')
```
with just a few deletions. no having to move arguments around

Reply:
> but then how do you differentiate between rgb and rgba? You need to know how many dims to pass to imshow so you can set up sliders for the rest of them.

oh yeah woops - good point. Hoever, I stand by moving them lower into the list of kwargs